src/tiles/item-image.ts

```diff
@@ -52,12 +56,29 @@ export class ItemImage extends LitElement {
   }
 
   // Templates
-  private get backgroundImageTemplate() {
+  private get itemImageTemplate() {
+    return html`
+      ${this.isListTile ? this.listImageTemplate : this.tileImageTemplate}
+    `;
+  }
+
+  private get tileImageTemplate() {
     return html`
       <div
         class=${this.imageClass}
         style="background-image:url(${this.imageSrc})"
       ></div>
+      ${this.tileActionTemplate}
+    `;
+  }
+
+  private get listImageTemplate() {
+    if (!this.model) {
+      return nothing;
+    }
+    return html`
+      <img src="${this.imageSrc}" alt="" class="${this.listImageClass}" />
```

Comment:
is alt still blank?

Reply:
other than that, LGTM

Reply:
Yes, we discussed this - not having a screen reader repeat the title for a user.

Reply:
We might do this for the other images, I.e., waveforms, line 91

Reply:
is it possible to have a generic name as `alt` in case?

Reply:
...hierarchy-wise, it would be nice to have the alt here instead of the image. bc when u source svg using img tag, the title may not come thru as opposed to having it on the img tag straight up.